app/RecordIssuerType.php

```diff
@@ -6,5 +6,12 @@
 
 class RecordIssuerType extends Model
 {
-    //
+  // record issuer type name stored in database
+  const BILL_TYPE_NAME =  'billing organization';
+  const BANK_STATEMENT_TYPE_NAME = 'bank';
+
+  public function record_issuer()
+  {
+    $this->hasMany(UserRecordIssuer::class);
```

Comment:
does not work that way  , has many through is for many to many through pivot table. issuerType and issuer is one to many.
but it sitll  has bug thanks for pointing out  because  i forgot to 'return' $this->